### PR TITLE
improved tex & python w/ vim-python/python-syntax

### PR DESCRIPTION
**Pull request by K1DV5**
I tried to solve the question "Why does file syntax not look exactly like in Visual Studio Code?" for python and latex. The python part needs the vim plugin python-syntax.

### CODE REVIEW

File: colors/codedark.vim [hunk 1]
```diff
@@ -113,7 +113,7 @@ let s:cdBlue = {'gui': '#569CD6', 'cterm': s:cterm0D, 'cterm256': '75'}
 let s:cdDarkBlue = {'gui': '#223E55', 'cterm': s:cterm0D, 'cterm256': '73'}
 let s:cdLightBlue = {'gui': '#9CDCFE', 'cterm': s:cterm0C, 'cterm256': '117'}
 if g:codedark_conservative | let s:cdLightBlue = s:cdFront | endif
-let s:cdGreen = {'gui': '#608B4E', 'cterm': s:cterm0B, 'cterm256': '65'}
+let s:cdGreen = {'gui': '#6A9955', 'cterm': s:cterm0B, 'cterm256': '65'}
```
Comment (tomasiser) [inline colors/codedark.vim]: Why did you change this color? :)

File: colors/codedark.vim [hunk 2]
```diff
@@ -189,10 +189,9 @@ call <sid>hi('Repeat', s:cdPink, {}, 'none', {})
 call <sid>hi('Label', s:cdPink, {}, 'none', {})
 call <sid>hi('Operator', s:cdFront, {}, 'none', {})
 call <sid>hi('Keyword', s:cdPink, {}, 'none', {})
-call <sid>hi('pythonOperator', s:cdPink, {}, 'none', {})
 call <sid>hi('Exception', s:cdPink, {}, 'none', {})
 
-call <sid>hi('PreProc', s:cdPink, {}, 'none', {})
+call <sid>hi('PreProc', s:cdBlue, {}, 'none', {})
```
Comment (tomasiser) [inline colors/codedark.vim]: Not sure with this change, because preproc is sometimes pink in VS Code. :(

Comment (tomasiser) [reply colors/codedark.vim]: Would it be possible to add some kind of condition if this is Python/LaTeX, switch preproc to blue perhaps?

Comment (K1DV5) [reply colors/codedark.vim]: Actually that is not important, I just changed it because it looked better on the vim help files, nothing important. It can be discarded.

File: colors/codedark.vim (hunk 1 above)
Comment (K1DV5) [reply colors/codedark.vim]: Because when I used a color picker to get the comment color in vscode, I got this value.

**Comment by tomasiser** [comment]
Hi @K1DV5, if you are still around, I would like to merge this PR, but could you please remove the two changes (lines 116 and 194) before I merge? I cannot do it myself as this PR is based on your fork.

An additional question: is the `if exists('g:python_highlight_builtin_funcs_kwarg')` necessary?

Thanks!

**Comment by K1DV5** [comment]
Hi, I'm here, will do.

And about the check, I agree, I have removed it on my fork already, but I haven't pushed it yet (it has many more modifications). Will remove that too.

**Comment by tomasiser** [comment]
Thanks! Merged!
